src/api/modules/billingMessages.ts

```diff
@@ -0,0 +1,21 @@
+import chalk from 'chalk'
```

Comment:
Since this file is not being used by other files, I think that he don't belong to `/api` folder. How about moving to `/src/lib/constant/`

Reply:
Makes sense, done in c3a67717ae2ec0c3ee9d60efbd6a701ab487cc86 !